tools/coqdep/lib/lexer.mll

```diff
@@ -120,14 +120,10 @@ and extra_dep_rule from = parse
 and require_modifiers from = parse
   | "(*"
       { comment lexbuf; require_modifiers from lexbuf }
-  | "Import" space+
+  | "Import" space*
```

Comment:
I'm not sure about allowing no spaces between the `Import` and `-`. @SkySkimmer is this something we want to do?

Reply:
That's how Coq syntax works